src/components/tabs/Tabs.vue

```diff
@@ -2,8 +2,8 @@
   <div>
     <div class="aui-tabs horizontal-tabs">
       <ul class="tabs-menu">
-        <li ref="item" class="menu-item" v-for="tab in tabs">
-          <a :href="'#' + tab.componentOptions.propsData.tabId">{{tab.data.attrs.name}}</a>
+        <li class="menu-item" :key="tab.hash" v-for="tab in tabs">
+          <a :ref="'tab_link_' + tab.hash" :href="'#' + tab.hash" v-html="tab.name"></a>
```

Comment:
It would great if you provided an example in the docs or using rich content via `v-html`. Probably adding a badge or sth sounds like a real case scenario. What are you using it for?

Reply:
`:ref="'tab_link_' + tab.hash"` can be replaced with nicer (I believe)
``:ref="`tab_link_${tab.hash}`"``

Reply:
Please check my last commit a6357b2